app/views/admin/organizations/index.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :title do
+  - 'Co-ops and Organizations'
```

Comment:
Should the link be coops to avoid confusion between coops and lending organizations?